LISA/LISAJsonRpc.cpp

```diff
@@ -233,11 +300,11 @@ namespace Plugin {
                 return errorCode;
             });
 
-        // CancelParamsInfo is used instead of GetProgressParamsInfo which 
```

Comment:
Could use the same trick as for GetMetadataParamInfo comment above.

Reply:
Done